DEVELOPING_TUTORIALS.md

```diff
@@ -639,48 +562,40 @@ Note that your user history on the live website (https://proto.school) is differ
 
 ### Renaming a tutorial after it has been published
 
-On rare occassion, we may need to change the name of a tutorial, and its related URL, after it has been published. Because we use IDs as our primary point of reference and have built a migration tool for the cache, it's possible to do this without affecting the user's progress indicators.
+1. In the `src/static/tutorials.json` file, rename `title` to the new desired title.
+1. Add old tutorial `url` value to the `redirects` attribute. (e.g. `"redirects": ["old-tutorial-short-title"]`)
+1. Update `url` with the new name.
+1. Rename folder `src/tutorials/xxxx-old-tutorial-short-title` to the new url. (e.g. `src/tutorials/xxxx-new-tutorial-short-title`)
 
```

Comment:
Do they not need to rename all the individual files that include the old shortname?

Reply:
the lesson files?